.eslintrc

```diff
@@ -96,6 +96,7 @@
       ],
       "rules": {
         "oppia/e2e-action": "error",
+        "oppia/no-then-method": "error"
```

Comment:
This should not be here as it is not covering lots of e2e test files. (I think you can remove this line after moving the checks to protractor-practices.)

Reply:
Done!